ftw/keywordwidget/configure.zcml

```diff
@@ -83,4 +83,12 @@
         directory="./upgrades/select2js"
         />
 
+    <browser:page
+        for="z3c.form.interfaces.ISelectWidget"
```

Comment:
Should I add a seperate interface for the keywordwidget? Probably useful in the future.